yolo/tools/drawer.py

```diff
@@ -13,7 +13,7 @@
 
 def draw_bboxes(
     img: Union[Image.Image, torch.Tensor],
```

Comment:
Why not use the union operator instead?
```suggestion
    img: Image.Image | torch.Tensor,
```

https://peps.python.org/pep-0604/

I suppose you are aiming to support older versions than python 3.10 right?